subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -3929,11 +3929,6 @@ static void socket_receive_loop(void)
 			len = recvfrom(sock_ctx[i]->sock_fd, in_buf,
 				       sizeof(in_buf) - 1, 0,
 				       &from_addr, &from_addr_len);
-			if (errno) {
```

Comment:
I personally would explicitly mention in the commit message that there's already check in the code for `len < 0` case.